File: cpp/raindrops/raindrops.cpp
#include "raindrops.h"
#include <string>

namespace raindrops {
    std::string convert(int drop) {
        std::string result = "";

        if (drop % 3 == 0) {
            result.append("Pling");
        }
        if (drop % 5 == 0) {
            result.append("Plang");
        }
        if (drop % 7 == 0) {
            result.append("Plong");
        }
        if (result.length() == 0){
            result = std::to_string(drop);
        }

        return result;
    }
}  // namespace raindrops

File: python/raindrops/raindrops.py
def convert(number):
    result = ""
    if not number % 3:
        result += "Pling"
    if not number % 5:
        result += "Plang"
    if not number % 7:
        result += "Plong"
    if len(result) < 1:
        result += str(number)
    return result